### ISSUE TITLE
Maven native build proposal

### ISSUE THREAD

**Issue by charizardmatok** [manuelbl/SwissQRBill#93]
Could the library be adapted to work in maven native build?
Currently it's not working due to java.awt not being fully supported:
`org.jboss.resteasy.spi.UnhandledException: java.lang.InternalError: java.lang.ClassNotFoundException: sun.awt.X11FontManager`

**Comment by manuelbl** [comment]
Do you mean a build for GraalVM?

The backend of the demo application for the library (https://github.com/manuelbl/SwissQRBillDemo/tree/master, published at https://www.codecrete.net/qrbill/) actually uses a native build. It is built on top of the Quarkus framework.

AWT is known to be a challenge with GraalVM. The Quarkus people built a special addition that takes care of it. In the demo application, it's configured here: https://github.com/manuelbl/SwissQRBillDemo/blob/master/service/pom.xml#L54

If you are also using Quarkus, that should help. If you don't use it, the extension might help you to understand what's needed to get it running.

**Comment by charizardmatok** [comment]
Exactly, GraalVM build. I am running quarkus also. I believe I once tried to use quarkus-awt dependency to bypass this, but it resulted in the same error. I will try it again just for good measure.

**Comment by charizardmatok** [comment]
Yes, I understand that, and I tried it following the changes made in the provided commit, but I am still getting an issue with a bunch of classes being inited at build time instead of runtime. 
Error: Classes that should be initialized at run time got initialized during image building: ...
